content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: influxctl
+list_title: influxctl
+description: >
+  The `influxctl` command line interface (CLI) performs administrative tasks in
+  an InfluxDB Cloud Dedicated cluster.
+menu:
+  influxdb_cloud_dedicated:
+    name: influxctl
+    parent: CLIs
+weight: 101
+influxdb/cloud-dedicated/tags: [cli]
+---
+
+The `influxctl` command line interface (CLI) performs administrative tasks in
+an InfluxDB Cloud Dedicated cluster.
+
+- [Usage](#usage)
+- [Commands](#commands)
+- [Flags](#command-flags)
+- [Download and install influxctl](#download-and-install-influxctl)
+- [Configure connection profiles](#configure-connection-profiles)
+- [Authentication](#authentication)
+
+## Usage
+
+```sh
+influxctl [flags] [command]
+```
+
+## Commands
+
+| Command                                                                 | Description                            |
+| :---------------------------------------------------------------------- | :------------------------------------- |
+| [database](/influxdb/cloud-dedicated/reference/cli/influxctl/database/) | Manage InfluxDB databases              |
+| [token](/influxdb/cloud-dedicated/reference/cli/influxctl/token/)       | Manage InfluxDB database tokens        |
+| [version](/influxdb/cloud-dedicated/reference/cli/influxctl/version/)   | Output the current `influxctl` version |
+| [help](/influxdb/cloud-dedicated/reference/cli/influxctl/help/)         | Output `influxctl` help information    |
+
+## Flags {#command-flags}
+
+| Flag        | Description                                                |
+| :---------- | :--------------------------------------------------------- |
+| `--profile` | Specify a connection profile to use (default is `default`) |
+
+---
+
+## Download and install influxctl
+
+{{< tabs-wrapper >}}
+{{% tabs %}}
+[Linux](#)
+[macOS](#)
+[Windows](#)
+{{% /tabs %}}
+{{% tab-content %}}
+<!-------------------------------- BEGIN Linux -------------------------------->
+
+<!-- TODO: Linux installation instructions -->
+
+<!--------------------------------- END Linux --------------------------------->
+{{% /tab-content %}}
+{{% tab-content %}}
+<!-------------------------------- BEGIN macOS -------------------------------->
+
+<!-- TODO: macOS installation instructions -->
+
+<!--------------------------------- END macOS --------------------------------->
+{{% /tab-content %}}
+{{% tab-content %}}
+<!-------------------------------- BEGIN Windows ------------------------------->
+
+<!-- TODO: Windows installation instructions -->
+
+<!--------------------------------- END Windows -------------------------------->
+{{% /tab-content %}}
+{{< /tabs-wrapper >}}
+
+## Configure connection profiles
+
+To connect with your InfluxDB Cloud Dedicated cluster, `influxctl` needs the
+following credentials:
+
+- InfluxDB Cloud Dedicated account ID
+- InfluxDB Cloud Dedicated cluster ID
+- InfluxDB Cloud Dedicated cluster URL
```

Comment:
I've removed cluster URL from the current README, but I won't know for sure till I get access to a cluster to try it all. My understanding right now is we only talk to granite directly and so we do not need to know the URL